trigger-openqa_in_openqa

```diff
@@ -21,7 +21,8 @@ machine="${machine:-"64bit"}"
 build_tag=${BUILD_TAG:-}
 client_prefix=${client_prefix:-}
 full_run=${FULL:-}
-group_id="${group_id:-"openQA"}"
+group_id=${group_id:-"openQA"}
+staging_group_id=${staging_group_id:-"openQA Staging"}
```

Comment:
I'm not sure whether I'd call the group "openQA Staging". The purpose of the contained jobs is to make submissions to Factory. So maybe "Factory submissions"? (That an intermediate repo is used is more an implementation detail which of course could be mentioned in the group description.)

Reply:
Other opinions?

Reply:
One thing I noticed - we would probably call the group "Factory submissions" without the openQA prefix.
If we pass this as `_GROUP` then it could potentially be ambiguous in he future, if there is ever another group named like that.
Should we have a feature to be able to pass an unambiguous job group by adding the parent as well, like `_GROUP="openQA : Factory submissions"? (and then forbid colons in group names)

I will rename the group now, as there were no other suggestions.